lldb/source/Plugins/ABI/LoongArch/ABISysV_loongarch.cpp

```diff
@@ -644,32 +644,22 @@ void ABISysV_loongarch::AugmentRegisterInfo(
     std::vector<lldb_private::DynamicRegisterInfo::Register> &regs) {
   lldb_private::RegInfoBasedABI::AugmentRegisterInfo(regs);
 
+  static const std::unordered_map<std::string, std::string> reg_aliases = {
```

Comment:
A nit: perhaps call it `isa_to_abi_alias_map`; r0-r31 is ISA convention and a/s/t is ABI convention. (I think mostly ABI convention is favored in discussions about LA64 and RV64)